goth/runner/container/compose.py

```diff
@@ -82,7 +93,9 @@ def __init__(
         self._log_monitors = {}
         self._network_gateway_address = ""
 
-    async def start_network(self, log_dir: Path, force_build: bool = False) -> None:
+    async def start_network(
+        self, log_dir: Path, force_build: bool = False
+    ) -> Dict[str, ContainerInfo]:
         """Start the compose network based on this manager's compose file.
 
         This step may include (re)building the network's docker images.
```

Comment:
Let's mention the newly-added return value here.

Reply:
Done (318fb5fa0)